code/game/gamemodes/demon/true_demon/_true_demon.dm

```diff
@@ -0,0 +1,191 @@
+#define DEVIL_HANDS_LAYER 1
+#define DEVIL_HEAD_LAYER 2
+#define DEVIL_TOTAL_LAYERS 2
+
+
+/mob/living/carbon/true_devil
```

Comment:
You can ban him from other human slots in species datum. Or make him only able to wear specific items in these slots. That's easy. Easier than managing yet another ultra special carbon.